### PR TITLE
Added support for "jpg" base64 encoded images

### PR DESCRIPTION
**Pull request by FelixFrog**
Previously this would create problems for certain svgs, making image not render

### CODE REVIEW

**Comment by deeplook** [comment]
Does it make sense to add "jpeg" (with an "e") as well?

File: svglib/svglib.py [hunk 1]
```diff
@@ -685,7 +685,7 @@ def xlink_href_target(self, node, group=None):
             return None
 
         # First handle any raster embedded image data
-        match = re.match(r"^data:image/(jpeg|png);base64", xlink_href)
+        match = re.match(r"^data:image/(jpeg|jpg|png);base64", xlink_href)
```
Comment (claudep) [inline svglib/svglib.py:688]: As we are in a regex already, I would simply put a `?` after the `e`.

Comment (FelixFrog) [reply svglib/svglib.py:688]: Yes, you're right

**Comment by FelixFrog** [comment]
> Does it make sense to add "jpeg" (with an "e") as well?

I think the code already works for ```href="data:image/jpeg;base64,..."``` images. It just didn't process the ones where jpg is spelled without and "e", resulting in incomplete renders.

**Comment by claudep** [comment]
Thanks!

**Pull request merged**